lib/matplotlib/tests/test_axes.py

```diff
@@ -2261,6 +2261,24 @@ def test_errorbar_limits():
     ax.set_title('Errorbar upper and lower limits')
 
 
+@image_comparison(baseline_images=['errorbar_offsets'],
+                  extensions=['png', 'svg'])
+def test_errorbar_offsets():
+    x = np.linspace(0, 1, 15)
+    y = x * (1-x)
+    yerr = y/6
+    fig = plt.figure()
+    ax1 = fig.add_subplot(2, 1, 1)
+    ax2 = fig.add_subplot(2, 1, 2)
+    for shift in [0, 0, 2, 3]:
```

Comment:
Are there supposed to be two 0s in this test, not just `range(4)`?

Reply:
Perhaps put a shift > N to test the start%N vs start ambiguity mentioned above.

Reply:
comment still applies